app/models/consultation.rb

```diff
@@ -49,8 +49,11 @@ class AddNeighbourAddressesError < StandardError; end
     has_many :neighbour_responses
   end
 
-  has_many :reviews, as: :owner, dependent: :destroy
-  has_many :neighbour_reviews, -> { where(specific_attributes: {consultation_type: "neighbour"}) }, as: :owner, class_name: "Review", dependent: :destroy
+  with_options as: :owner do
+    has_many :reviews, dependent: :destroy
+    has_many :neighbour_reviews, -> { neighbour_reviews }, class_name: "Review" # rubocop:disable Rails/HasManyOrHasOneDependent
+    has_one :neighbour_review, class_name: "Review" # rubocop:disable Rails/HasManyOrHasOneDependent
```

Comment:
Hmm, not sure whether there's something about `dependent: :destroy` I'm missing or rubocop's rule is a bit braindead - going to have to have a look at this now for my own sanity.